homeassistant/helpers/service.py

```diff
@@ -2,6 +2,10 @@
 import functools
 import logging
 
+# pylint: disable=unused-import,ungrouped-imports
+from typing import Optional  # NOQA
+from homeassistant.helpers.typing import HomeAssistantType  # NOQA
```

Comment:
Cant we make `HomeAssistantType` Optional by default? This seems like three hoops to jump through to get one Type (since it might be used in more places, bootstrap.py to think of one)

Or add a `oHomeAssistantType` or `OptionalHomeAssistantType`?